doc/docs/flymd.md

```diff
@@ -0,0 +1,47 @@
+---
+# NLopt on Windows
+---

```

Comment:
Did this file get included accidentally?

Reply:
You're right it happened accidentally.